tactics/tacticals.mli

```diff
@@ -115,6 +115,8 @@ val tclMAPDELAYEDWITHHOLES : bool -> 'a delayed_open list -> ('a -> unit tactic)
     argument of [l] are evaluated in the possibly-updated
     environment and updated sigma of each new successive goals *)
 
+val check_evar_list : Environ.env -> evar_map -> Evar.Set.t -> evar_map -> (?loc:Loc.t -> Evar.t -> unit) -> unit
```

Comment:
Although I already validated, one last nitpick, I swear: this HO API is bad as it's unclear how the continuation is going to be used, and furthermore it forces to raise an exception due to the unit return type. Instead of doing this CPS, you should instead return an `Evar.t option`.

Reply:
Yes, better, done (returning an `Evar.t list`).